docs/usage.rst

```diff
@@ -120,6 +120,15 @@ pseudo-elements::
     >>> selector.css('title::text').get()
     'Example website'
 
+Extract text witout ::text
+==========================
```

Comment:
There should not be a header here, it splits the existing content in a weird way (not everything from this point to the next header belongs to the header).

Reply:
Heading has been removed